code/game/objects/items/devices/radio/radio.dm

```diff
@@ -307,6 +307,10 @@
 	if(radio_freq || !broadcasting || get_dist(src, speaker) > canhear_range)
 		return
 
+	var/filtered_mods = list()
```

Comment:
Don't init a list unless it's needed.

Reply:
it gets passed around in procs that assume the list is initialized and not null